Description: Check if schema is defined for volume slider changes #1544
Author: David Mohammed <fossfreedom@ubuntu.com>
Commit: https://github.com/UbuntuBudgie/budgie-desktop/commit/42565ccebf0cc0da007e0127a4428b015e59fb07
Last Update: 2018-08-25

---
 src/raven/sound.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/raven/sound.vala b/src/raven/sound.vala
index b9a514e3..a7971b6a 100644
--- a/src/raven/sound.vala
+++ b/src/raven/sound.vala
@@ -421,7 +421,7 @@ namespace Budgie {
             var vol = primary_stream.get_volume();
             var vol_max = mixer.get_vol_max_norm();

-            if (settings.get_boolean(MAX_KEY)) { // Allowing max
+            if (settings != null && settings.get_boolean(MAX_KEY)) { // Allowing max
                 vol_max = mixer.get_vol_max_amplified();
             }

--
2.17.1

